configuration/config2/cfg20.sln

```diff
@@ -0,0 +1,31 @@
+﻿
```

Comment:
no IDE specific code

Reply:
right, if you get cmake right you can generate files for VS.